src/Components/Users/UserProfile.tsx

```diff
@@ -186,9 +187,33 @@ export default function UserProfile() {
         }
     };
 
-    return (
+    const checkForNewBuildVersion = async () => {
+
+        let [isLatestBundle, newVersion] = await checkIfLatestBundle();
+
+        if (!isLatestBundle) {
+            setUpdateBtnText("updating...");
+            localStorage.setItem('build_meta_version', newVersion);
```

Comment:
Codacy found an issue: [localStorage can be unsafe](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5837957)